.github/workflows/build.yaml

```
@@ -0,0 +1,17 @@
+name: build and test
+
+on: [ push ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
```

````suggestion
      - name: Install Java and Maven
        uses: actions/setup-java@v3.3.0
        with:
          distribution: 'zulu'
          java-version: '11'
````

We should use the same version of the action all workflows